src/app/components/ChannelList/ChannelRow.tsx

```diff
@@ -4,6 +4,8 @@ import Identicon from 'components/Identicon';
 import Unit from 'components/Unit';
 import { ChannelWithNode } from 'modules/channels/types';
 import './ChannelRow.less';
+import {Progress} from 'reactstrap';
+import './custom-progress.css';
```

Comment:
All styles in src should use LESS rather than CSS.

Reply:
fixed